templates/planning/planning-weeks.html.twig

```diff
@@ -189,7 +194,7 @@
                                         {% set res = null %}
```

Comment:
if res should be a number I think you should start with a value of 0 instead of null

Reply:
Settings res to 0 by default causes all unset cells to contain a "0", which Line deemed annoying so we went with null. I added a comment describing this.

Reply:
another option could be

`{{ res != 0 ? res : '' }}` 

instead of

`{{ res }}`